english/explore-outside/talk-child/talk-more/nice-talk/walk-away/walk-away.md

```diff
@@ -0,0 +1,2 @@
+Unsure how to react, you decide to simply walk away from this dilemma.
+As you continue to explore this strange world, you stumble across
```

Comment:
Was it intentional that the story stops mid-sentence? I'd rather that instead, you delete this sentence, leaving the reader simply walking away, or that you specify what the reader stumbles across and let the next person to continue the story determine the reaction. That way, until the next person adds to this path, the story will still make sense.